hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/DoubleArrayCost.java

```diff
@@ -106,4 +88,32 @@ private static double getSum(double[] stats) {
     }
     return total;
   }
+
+  /**
+   * Return the min skew of distribution
+   */
+  public static double getMinSkew(double total, double numServers) {
```

Comment:
Is `total` the "total number of regions in the cluster"?

Reply:
yes.

Reply:
nit: one way of addressing the nick comment would be to add javadoc for total that said what it was.....